net-mail/cyrus-imapd/cyrus-imapd-3.8.5.ebuild

```diff
@@ -0,0 +1,277 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Keep an eye on https://www.cyrusimap.org/imap/developer/compiling.html!
+inherit autotools eapi9-ver flag-o-matic pam ssl-cert
+
+DESCRIPTION="The Cyrus IMAP Server"
+HOMEPAGE="https://www.cyrusimap.org/"
+SRC_URI="https://github.com/cyrusimap/${PN}/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="BSD-with-attribution GPL-2"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="
+	afs backup calalarm caps clamav http kerberos ldap
+	mysql nntp pam perl postgres replication
+	ssl static-libs tcpd test xapian
+"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	afs? ( kerberos )
+	calalarm? ( http )
+	http? ( xapian )
+"
+
+COMMON_DEPEND="
+	dev-libs/libpcre:=
```

Comment:
for versions of 3.8.x >=3.8.2 the code supports pcre2 perhaps this should be updated to that depend (and the configure)

Reply:
Upstream supports it from 3.8 and up, but they changed the default in 3.10 and 3.12. So I followed that.

Reply:
libpcre2 was added as experimental in 3.4.7, from the versions I mentioned (3.6.4 and 3.8.2), it is no longer marked as experimental.
I'm not suggesting making it a USE flag option, hard depend on libpcre2 where it's marked stable i.e. the 3.6 and beyond ebuilds ( libpcre:3 did at one point cause namespace collisions with libc 'regcomp' - https://github.com/cyrusimap/cyrus-imapd/issues/1731 ) once bitten twice shy.

3.4.9 is running fine here (prod on x86 and testing on amd64) with libpcre2 (I am NOT suggesting adding libpcre2 dep to 3.4.x)